ParkPlanner/Pages/Index.cshtml

```diff
@@ -10,17 +10,26 @@
     <h1 class="display-4">Welcome to Park Place</h1>
     <h2>Please stay tuned. Our site is currently under construction.</h2>
     <img src="https://user-images.githubusercontent.com/88552005/277193898-3a55302a-d57f-4ee4-86c3-15473b3e651e.jpg" alt="Park Planner" width="500">
-    <table>
-        <tr>
-            <th>Park ID</th>
-        </tr>
-        @{
-            foreach (Datum parks in park)
-            {
+   @if (park != null && park.Any())
```

Comment:
While the foreach will handle empty collections, no problem, there is an advantage to wrapping all of this in an if test.  It prevents the table header from being printed when there are no results.